2-mandatory/4-people-i-know.js

```diff
@@ -394,6 +402,15 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
+friends.forEach((item) => {
+  for (const friend of item.colleagues) {
+    if (friend.name === "Stacie Villarreal") {
```

Comment:
Very small thing, the naming of the varibles here can be confusing. I would rename item to friend and friend to colleague, so they match the array they come from